docs/fundamentals/connection/connection-options.txt

```diff
@@ -349,3 +350,40 @@ item, as shown in the following example:
 
 See the `$driverOptions: array <https://www.mongodb.com/docs/php-library/current/reference/method/MongoDBClient__construct/#parameters>`__
 section of the {+php-library+} documentation for a list of driver options.
+
+.. _laravel-disable-id-alias:
+
+Disable Use of id Field Name Conversion
+---------------------------------------
```

Comment:
```suggestion
Disable `id` to `_id` Field Name Conversion
-------------------------------------------
```

Reply:
The documentation style guide advises against font treatments in headings - [link](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#:~:text=Don%27t%20apply%20font%20treatments%20(bold%2C%20italics%2C%20or%20monospace)%20to%20text%20in%20a%20title%20or%20heading.)